yarn.lock

```diff
@@ -76,7 +76,7 @@
     "@algolia/requester-common" "4.15.0"
     "@algolia/transporter" "4.15.0"
 
-"@algolia/client-search@>= 4.9.1 < 6", "@algolia/client-search@4.15.0":
```

Comment:
this is a huge changelog for a simple library addition. can u verify if this yarn lock change is necessary ?

Reply:
I don't have idea why this changes :S

Reply:
reset it to main . then add the dependent libray